app/helpers/gems_helper.rb

```diff
@@ -0,0 +1,2 @@
+module GemsHelper
+end
```

Comment:
Same here, I think it's safe to delete this file too since we're not really using the `GemsHelper`.